cpp/oneapi/dal/algo/kmeans/infer_types.cpp

```diff
@@ -100,9 +97,6 @@ double infer_result<Task>::get_objective_function_value() const {
 
 template <typename Task>
 void infer_result<Task>::set_responses_impl(const table& value) {
-    if (!get_result_options().test(result_options::compute_assignments)) {
```

Comment:
please remove it as well and update the default behavior

Reply:
done